includes/wf_crm_webform_postprocess.inc

```diff
@@ -2178,6 +2180,13 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
         elseif ($field['type'] === 'number') {
           $sum = 0;
           foreach ((array) $val as $k => $v) {
+            //Strip any thousand separator for further calculation.
+            $separator = wf_civicrm_api('Setting', 'getvalue', array(
+              'name' => "monetaryThousandSeparator",
+            ));
+            if ($field['data_type'] === 'Money') {
+              $v = preg_replace("/(?<=\d){$separator}(?=\d{3}\b)/", '', $v);
+            }
```

Comment:
I haven't tried it out with alternate thousands separators but I'd imagine this will also require fixing the decimal place symbol.